aspnetcore/performance/caching/memory.md

```diff
@@ -49,9 +49,9 @@ Use `System.Runtime.Caching`/`MemoryCache` as a compatibility bridge when portin
 ## Use IMemoryCache
 
 > [!WARNING]
-> Using a *shared* memory cache from [Dependency Injection](xref:fundamentals/dependency-injection) and calling `SetSize`, `Size`, or `SizeLimit` to limit cache size can cause the app to fail. When a size limit is set on a cache, all entries must specify a size when being added. This can lead to issues since developers may not have full control on what uses the shared cache. For example, Entity Framework Core uses the shared cache and does not specify a size. If an app sets a cache size limit and uses EF Core, the app throws an `InvalidOperationException`.
+> Using a *shared* memory cache from [Dependency Injection](xref:fundamentals/dependency-injection) and calling `SetSize`, `Size`, or `SizeLimit` to limit cache size can cause the app to fail. When a size limit is set on a cache, all entries must specify a size when being added. This can lead to issues since developers may not have full control on what uses the shared cache.
 > When using `SetSize`, `Size`, or `SizeLimit` to limit cache, create a cache singleton for caching. For more information and an example, see [Use SetSize, Size, and SizeLimit to limit cache size](#use-setsize-size-and-sizelimit-to-limit-cache-size).
-> A shared cache is one shared by other frameworks or libraries. For example, EF Core uses the shared cache and does not specify a size. 
+> A shared cache is one shared by other frameworks or libraries. For example, EF Core uses the shared cache and specifies a size.
```

Comment:
By default it doesn't use the shared memory cache from DI anymore. You need to very intentionally force it to use the shared cache, so I think this still isn't right. 

cc @ajcvickers

Reply:
@divega is right; I missed that. By default, EF Core doesn't interact with this memory cache at all.